src/util/RandomLB.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AbstractLoadBalancer} from './AbstractLoadBalancer';
+import {Member} from '../core/Member';
+import {randomInt} from '../Util';
+
+/**
+ * A {@link LoadBalancer} that selects a random member to route to.
+ */
+export class RandomLB extends AbstractLoadBalancer {
```

Comment:
nit: do we use this class somewhere?

Reply:
No, I thought it may be used by the user as an alternative load balancer. We may remove it if it seems unnecessary.

Reply:
I'd rather remove it. But if you're like to keep it, it makes sense to document it.

Reply:
Added the documentation. Java client supports setting load balancer with declarative configuration, so I also added support for this. Also, I added the tests for the load balancers.

Reply:
Thanks for going an extra mile here. That's a valuable addition.